src/gui/editors/PianoRoll.cpp

```diff
@@ -2806,13 +2816,14 @@ void PianoRoll::dragNotes( int x, int y, bool alt, bool shift, bool ctrl )
 		else
 		{
 			// shift is not pressed; stretch length of selected notes but not their position
+			int minLength = alt ? 1 : m_minResizeLen.getTicks();
```

Comment:
How does a length of 1 handle move and resize actions?

Reply:
If alt is held we want to allow notes to be scaled to any length, so 1 tick is the minimum.

Reply:
> How does a length of 1 handle move 

I don't know if this varies with screen but on my small one it's not possible to get a hand icon when hovering over a 1 tick note.

> and resize actions?

It readily resizes like other notes but you can't do it on the smallest zoom levels.

Reply:
> 
> 
> > How does a length of 1 handle move
> 
> I don't know if this varies with screen but on my small one it's not possible to get a hand icon when hovering over a 1 tick note.

I recall the minimum draw size should be like 4 pixels... Hm.


> 
> > and resize actions?
> 
> It readily resizes like other notes but you can't do it on the smallest zoom levels.

Good to know.

Reply:
> I recall the minimum draw size should be like 4 pixels... Hm.

Top: beat note
Bottom: 1 tick note
![beatnotevs1tick](https://user-images.githubusercontent.com/6368949/82604822-27f56180-9bb5-11ea-8985-5a61d3a5813e.png)

Reply:
https://github.com/LMMS/lmms/blob/c3e056a21df4601f1d537b0719c42eff523c9935/src/gui/editors/PianoRoll.cpp#L2126-L2129

Set  `else if( len_ticks < 0 )` to  `else if( len_ticks < 4 )` to make 1 tick notes same length as beat notes. It won't improve ability to grab and stretch them however. It's also not related to this PR so </OT>  ;)